concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-914	914	Verfolgen	to pursue	VIII. Verba	D. Gehen, Kommen u.s.w.	1047	PURSUE
+Gabelentz-1892-1074-915	915	Erreichen	to arrive at a destination	VIII. Verba	D. Gehen, Kommen u.s.w.	1387	ARRIVE
+Gabelentz-1892-1074-916	916	Umkehren	to turn back	VIII. Verba	D. Gehen, Kommen u.s.w.	581	COME BACK
+Gabelentz-1892-1074-917	917	Schicken, senden	to send	VIII. Verba	D. Gehen, Kommen u.s.w.	1127	SEND
+Gabelentz-1892-1074-918	918	Eintreten	to enter	VIII. Verba	D. Gehen, Kommen u.s.w.	749	ENTER
+Gabelentz-1892-1074-919	919	Treten	to kick	VIII. Verba	D. Gehen, Kommen u.s.w.	1135	KICK
+Gabelentz-1892-1074-920	920	Laufen	to walk	VIII. Verba	D. Gehen, Kommen u.s.w.	1443	WALK
+Gabelentz-1892-1074-921	921	Kriechen	to crawl	VIII. Verba	D. Gehen, Kommen u.s.w.	3215	CREEP
+Gabelentz-1892-1074-922	922	Klettern	to climb	VIII. Verba	D. Gehen, Kommen u.s.w.	1102	CLIMB
+Gabelentz-1892-1074-923	923	Fliegen	to fly	VIII. Verba	D. Gehen, Kommen u.s.w.	1441	FLY (MOVE THROUGH AIR)
+Gabelentz-1892-1074-924	924	Fliehen	to flee	VIII. Verba	D. Gehen, Kommen u.s.w.	956	FLEE
+Gabelentz-1892-1074-925	925	Verlassen	to leave	VIII. Verba	D. Gehen, Kommen u.s.w.	1752	LEAVE
+Gabelentz-1892-1074-926	926	Aufgehen (die Sonne u.s.w.)	to rise (sun etc.)	VIII. Verba	D. Gehen, Kommen u.s.w.	568	RISE (MOVE UPWARDS)
+Gabelentz-1892-1074-927	927	steigen	to ascend	VIII. Verba	D. Gehen, Kommen u.s.w.	705	GO UP (ASCEND)
+Gabelentz-1892-1074-928	928	Untergehen (desgl.)	to set (sun etc.)	VIII. Verba	D. Gehen, Kommen u.s.w.	160	SET (HEAVENLY BODIES)
+Gabelentz-1892-1074-929	929	hinabsteigen	to descend	VIII. Verba	D. Gehen, Kommen u.s.w.	2014	GO DOWN (DESCEND)
+Gabelentz-1892-1074-930	930	Aufstehen	to get up	VIII. Verba	D. Gehen, Kommen u.s.w.	1512	RISE (ONESELF)
+Gabelentz-1892-1074-931	931	Schütteln	to shake	VIII. Verba	D. Gehen, Kommen u.s.w.	1909	SHAKE
+Gabelentz-1892-1074-932	932	sich schütteln	to shake (oneself)	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-933	933	Zittern	to tremble, to shiver	VIII. Verba	D. Gehen, Kommen u.s.w.	52	SHIVER
+Gabelentz-1892-1074-934	934	Hüpfen	to hop	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-935	935	Stampfen	to stomp	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-936	936	Springen	to jump	VIII. Verba	D. Gehen, Kommen u.s.w.	1829	JUMP
+Gabelentz-1892-1074-937	937	Tanzen	to dance	VIII. Verba	D. Gehen, Kommen u.s.w.	1879	DANCE
+Gabelentz-1892-1074-938	938	Straucheln	to stumble	VIII. Verba	D. Gehen, Kommen u.s.w.	3617	STUMBLE
+Gabelentz-1892-1074-939	939	Fallen	to fall	VIII. Verba	D. Gehen, Kommen u.s.w.	1280	FALL
+Gabelentz-1892-1074-940	940	Ertrinken	to drown	VIII. Verba	D. Gehen, Kommen u.s.w.	1904	DROWN
+Gabelentz-1892-1074-941	941	Schwimmen	to swim	VIII. Verba	D. Gehen, Kommen u.s.w.	1439	SWIM
+Gabelentz-1892-1074-942	942	Fliessen	to flow	VIII. Verba	D. Gehen, Kommen u.s.w.	2003	FLOW
+Gabelentz-1892-1074-943	943	Tröpfeln	to dribble	VIII. Verba	D. Gehen, Kommen u.s.w.		
```

Comment:
Maybe DRIP (FALL IN GLOBULES)?

Reply:
Good catch. I would even suggest the broader 3359 DRIP OR DROP SOMETHING, how about that?

Reply:
_tröpfeln_ is mapped to DRIP (EMIT LIQUID) in Gabelentz-1891-120-116 so I'd suggest to map it to this concept as well.

Reply:
I think it depends on whether it it's meant here transitively (and then it can be mapped to DRIP (EMIT LIQUID)) or intransitively (and then it is a case for DRIP (FALL IN GLOBULES)). If it's unclear, then I think it would make sense to map to more general DRIP OR DROP SOMETHING as suggested by Carolin.